api/v1beta1/keystoneendpoint.go

```diff
@@ -0,0 +1,200 @@
+/*
```

Comment:
this is just part of a split out of the func.go into keystoneapi.go, keystoneendpoint.go and keystoneservice.go